Makefile

```diff
@@ -0,0 +1,3 @@
+lint:
+	isort --check-only
+	flake8 .
```

Comment:
Let's go with GitHub scripts style instead of this `Makefile`. (See https://github.com/tomchristie/apistar/issues/13)

So:

Remove the Makefile, and and a `/scripts/lint` executable command. (Could be either shell script or python, but don't include an extension on the script, and ensure that it's executable)